sdk/formrecognizer/azure-ai-formrecognizer/migration-guide.md

```diff
@@ -0,0 +1,597 @@
+    DocumentField merchantNameField = receiptFields.get("MerchantName");
+    if (merchantNameField != null) {
+        if (DocumentFieldType.STRING == merchantNameField.getType()) {
+            String merchantName = merchantNameField.getValueString();
+            System.out.printf("Merchant Name: %s, confidence: %.2f%n",
+                merchantName, merchantNameField.getConfidence());
+        }
+    }
+
+    DocumentField merchantPhoneNumberField = receiptFields.get("MerchantPhoneNumber");
+    if (merchantPhoneNumberField != null) {
+        if (DocumentFieldType.PHONE_NUMBER == merchantPhoneNumberField.getType()) {
+            String merchantAddress = merchantPhoneNumberField.getValuePhoneNumber();
+            System.out.printf("Merchant Phone number: %s, confidence: %.2f%n",
+                merchantAddress, merchantPhoneNumberField.getConfidence());
+        }
+    }
+
+    DocumentField transactionDateField = receiptFields.get("TransactionDate");
+    if (transactionDateField != null) {
+        if (DocumentFieldType.DATE == transactionDateField.getType()) {
+            LocalDate transactionDate = transactionDateField.getValueDate();
+            System.out.printf("Transaction Date: %s, confidence: %.2f%n",
+                transactionDate, transactionDateField.getConfidence());
+        }
+    }
+
+    DocumentField receiptItemsField = receiptFields.get("Items");
+    if (receiptItemsField != null) {
+        System.out.printf("Receipt Items: %n");
+        if (DocumentFieldType.LIST == receiptItemsField.getType()) {
+            List<DocumentField> receiptItems = receiptItemsField.getValueList();
+            receiptItems.stream()
+                .filter(receiptItem -> DocumentFieldType.MAP == receiptItem.getType())
+                .map(documentField -> documentField.getValueMap())
+                .forEach(documentFieldMap -> documentFieldMap.forEach((key, documentField) -> {
+                    if ("Name".equals(key)) {
+                        if (DocumentFieldType.STRING == documentField.getType()) {
+                            String name = documentField.getValueString();
+                            System.out.printf("Name: %s, confidence: %.2fs%n",
+                                name, documentField.getConfidence());
+                        }
+                    }
+                    if ("Quantity".equals(key)) {
+                        if (DocumentFieldType.FLOAT == documentField.getType()) {
+                            Float quantity = documentField.getValueFloat();
+                            System.out.printf("Quantity: %f, confidence: %.2f%n",
+                                quantity, documentField.getConfidence());
+                        }
+                    }
+                }));
+        }
+    }
+}
+```
+#### Using a layout model
+- Recognize text, table structures and selection marks like radio buttons and check boxes, along with their bounding box
+coordinates, from documents.
```

Comment:
I would just say the differences here again. I think in Python we say something like "Differences between versions". I don't think we need to introduce stuff like in the readme.